### PR TITLE
Fix Broken Haddock Derivation

### PR DESCRIPTION
**Pull request by zeme-iohk**

### CODE REVIEW

File: nix/cells/plutus/packages/read-the-docs-site.nix [hunk 1]
```diff
@@ -5,7 +5,7 @@ let
 in
 
 pkgs.stdenv.mkDerivation {
-  name = "read-the-docs-site";
+  name = "aaaaaaaa__read-the-docs-site";
```
Comment (zeme-iohk) [inline nix/cells/plutus/packages/read-the-docs-site.nix:8]: So that cicero runs this first ... will be reverted before merging

Comment (michaelpj) [reply nix/cells/plutus/packages/read-the-docs-site.nix:8]: I don't think this makes any difference any more, we switched to only having one job and then nix schedules everything.

Comment (zeme-iohk) [reply nix/cells/plutus/packages/read-the-docs-site.nix:8]: Ok 👍

File: doc/read-the-docs-site/adr/index.rst [hunk 2]
```diff
@@ -30,5 +30,4 @@ The general process for creating an ADR is:
    :maxdepth: 1
    :titlesonly:
 
-   0001-record-architecture-decisions
-   0002-steppable-cek-machine
```
Comment (zeme-iohk) [inline doc/read-the-docs-site/adr/index.rst:34]: I will bring this back once https://github.com/input-output-hk/plutus/pull/4930#issuecomment-1301769289 has been settled

**Pull request merged**